blocks/testimonials/testimonials.css

```diff
@@ -0,0 +1,41 @@
+.testimonials-grid {
```

Comment:
These styles should all be scoped to `.testimonials`.

```
.testimonals {
 ....
} 
```

Reply:
done, good catch